firestore/src/common/settings.cc

```diff
@@ -18,13 +18,7 @@ const char kDefaultHost[] = "firestore.googleapis.com";
 
 std::string ToStr(bool v) { return v ? "true" : "false"; }
 
-std::string ToStr(int64_t v) {
```

Comment:
You can remove this overload entirely and just directly call `std::to_string` below.

Reply:
Done